README.md

```diff
@@ -24,6 +24,17 @@
 Models are trained on the preprocessed dataset which can be downloaded [here](https://drive.google.com/open?id=0B_voCy5O5sXMTFByemhpZllYREU).
 
 ## Local Set-Up
+### Local:
+- It is recommended to set this project up inside a virtual environment to keep dependencies separated and for easy debugging. Resources:
+    * [Python](https://realpython.com/python-virtual-environments-a-primer/#why-the-need-for-virtual-environments)
+    * [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
+- Clone this repository by using `git clone https://github.com/imskr/Plant_Disease_Detection.git` or clone your forked repository.
+- Move inside the directory `cd Plant_Disease_Detection`
+- Activate your virtual environment. (references in the first step)
```

Comment:
Give link to references

Reply:
The references are the first step in the local setup instructions, which has references to create and activate virtual environments.

Reply:
Remove `(references in the first step)`